app/repository/SqlLiteDatabase.ts

```diff
@@ -16,9 +16,11 @@ interface TableName {
  */
 export class SqlLiteDatabase {
   db: Database;
+  type: 'pg' | 'sqlite';
```

Comment:
This type is multiple times, could we extract it to `domain/types.ts`
```typescript
export type DatabaseType = 'pg' | 'sqlite';
```

Reply:
fixed